site/pages/docs/specification.mdx

```diff
@@ -50,7 +50,7 @@ A Mini App URL must have a FrameEmbed in a serialized form in the `fc:frame` met
 
 | Property              | Type   | Required | Description                              | Constraints                                  |
 |-----------------------|--------|----------|------------------------------------------|----------------------------------------------|
-| type                  | string | Yes      | Mini App name.                           | Max length 32 characters                     |
```

Comment:
Looking at packages/frame-core/src/schemas/embeds.ts looks like there's `view_token` as well? Let's use the same approach as in `primaryCategory`:

```
One of: `launch_frame`, `view_token`
```